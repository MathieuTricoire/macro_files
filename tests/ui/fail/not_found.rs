use macro_files::create;

fn main() {
    create!({ "README.md": readme });
}
