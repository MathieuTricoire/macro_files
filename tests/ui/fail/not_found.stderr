error[E0425]: cannot find value `readme` in this scope
 --> tests/ui/fail/not_found.rs:4:28
  |
4 |     create!({ "README.md": readme });
  |                            ^^^^^^ not found in this scope
